lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -677,23 +681,18 @@ final class GenericRole extends SemanticRole {
       return;
     }
 
-    // Assign one of three roles to the element: group, heading, text.
+    // Assign one of three roles to the element: group or text.
```

Comment:
This comment is now outdated?

```suggestion
    // Assign one of two roles to the element: group or text.
```